src/webview2/mod.rs

```diff
@@ -499,6 +499,15 @@ impl InnerWebView {
       }
     }
 
+    // Extension loading
+    unsafe {
+      if pl_attrs.browser_extensions_enabled {
+        if let Some(extension_path) = pl_attrs.extension_path {
+          Self::load_extensions(&webview, &extension_path)?;
+        }
+      }
+    }
```

Comment:
let's move the unsafe block a little bit below

```suggestion
    if pl_attrs.browser_extensions_enabled {
      if let Some(extension_path) = pl_attrs.extension_path {
        unsafe { Self::load_extensions(&webview, &extension_path)? };
      }
    }
```